simpleSequential/test_isTwoSetEqual.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from isTwoSetEqual import isTwoSetEqual
+
+class TestIsTwoSetEqual(unittest.TestCase):
+    def test_equalSetIntegerInput(self):
+        self.assertEqual(isTwoSetEqual({6, 9}, {9, 6}), True)
+
+    def test_notEqualSetIntegerInput(self):
```

Comment:
"non" or "not"? you are not consistence in different files. Sometimes you use "nonEqual", sometime you use "notEqual".

I think you should stick to "non". Fix this.